src/box/lua/net_box.lua

```diff
@@ -749,7 +761,8 @@ function remote_methods:_request(method, opts, format, stream_id, ...)
     local res, err = transport:perform_request(timeout, buffer, skip_header,
                                                return_raw, on_push, on_push_ctx,
                                                format, stream_id, method, ...)
-    if err then
+    -- see comment on 730 line
```

Comment:
Please, don't mention exact line numbers. The comment you refer to will stay on line 730 for quite short time most likely and this reference will become outdated.

Reply:
My bad, will fix it